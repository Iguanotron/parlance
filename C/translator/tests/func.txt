int func(int arg1, int arg2) {
    
}
