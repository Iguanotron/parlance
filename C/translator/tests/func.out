DECLARE FUNCTION func TYPE int
ARG arg1 TYPE int ARG arg2 TYPE int
BEGIN
    
END SCOPE
